medcat/utils/preprocess_snomed.py

```diff
@@ -63,6 +64,17 @@ def to_concept_df(self):
                     continue
                 else:
                     pass
+            if self.uk_drug_ext:
+                if "SnomedCT_UKDrugRF2_PRODUCTION" in paths[i]:
+                    concept_snapshot = "sct2_Concept_UKDGSnapshot"
+                    description_snapshot = "sct2_Description_UKDGSnapshot-en"
+                elif "SnomedCT_UKEditionRF2_PRODUCTION" in paths[i]:
+                    concept_snapshot = "sct2_Concept_UKEDSnapshot"
+                    description_snapshot = "sct2_Description_UKEDSnapshot-en"
+                elif "SnomedCT_UKClinicalRefsetsRF2_PRODUCTION" in paths[i]:
+                    continue
+                else:
+                    pass
```

Comment:
these lines aren't needed

Reply:
The first two serve an actual function. The loop (which starts on line 52) has further functionality below these `if` statements (until line 121); the `elif` and `continue` are stopping these from being executed).

And the `else:\n pass` seems to be just to keep things in line with the rest of the code within this file.

The same seems to be true for the other flagged changes.

Reply:
ah right yeah - I expanded up the code... so these functions are pretty rough looking, maybe consider using some module level funcs etc. 

 else: pass, should be deleted everywhere, rather than adding more code that does nothing ;D 

just noticed - above the changed lines, the first `if` resets the default args, which also aren't needed:
<pre> 
            concept_snapshot = "sct2_Concept_Snapshot"
            relationship_snapshot = "sct2_Relationship_Snapshot"
            if self.uk_ext:
                if "SnomedCT_InternationalRF2_PRODUCTION" in paths[i]:
                    concept_snapshot = "sct2_Concept_Snapshot"
                    relationship_snapshot = "sct2_Relationship_Snapshot"
</pre>